sensor/kubernetes/sensor/sensor.go

```diff
@@ -151,9 +148,10 @@ func CreateSensor(cfg *CreateOptions) (*sensor.Sensor, error) {
 		complianceService,
 	}
 
-	vmService := virtualmachine.NewService()
+	var vmComponent virtualmachine.Component
```

Comment:
If there is going to be only one component for VMVM, then maybe we could call it more precisely - I think more towards `virtualmachine.ScanHandler` or `.Manager`? 
Using the word `Component` may overload it a bit, since we use Components as a generic term in Sensor for any component that can be started, stopped, etc.

Reply:
Sounds cool when concat to the pkg name. Thanks!